### PR TITLE
feat: add kemono.party support

### PR DESCRIPTION
**Pull request by y-young**

### CODE REVIEW

File: docs/site/kemono.zh.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Kemono.party
+
+<https://kemono.party/>，<https://kemono.su/>
+
+## 自定义存储路径和文件名
+
+更多信息请查阅 [自定义存储路径和文件名](./index.zh.md/#customizing-storage-path--file-name)。
+
+### LOFTER_FILE_PATH
```
Comment (Misaka13514) [inline docs/site/kemono.zh.md:9]:
```suggestion
### KEMONO_FILE_PATH
```

File: docs/site/kemono.zh.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+:material-lightbulb-on: 可选，默认为 `Kemono/{service}/{username} ({user})/{title} ({id})`
+
+存储路径。
+
+### LOFTER_FILE_NAME
```
Comment (Misaka13514) [inline docs/site/kemono.zh.md:15]:
```suggestion
### KEMONO_FILE_NAME
```

**Comment by Misaka13514** [comment]
maybe update `.env.example` as well? tell me if you need a pull request.

**Comment by y-young** [comment]
> maybe update `.env.example` as well? tell me if you need a pull request.

Feel free to open a PR.

**Pull request merged**